src/vulkan-renderer/imgui.cpp

```diff
@@ -10,13 +10,9 @@
 
 namespace inexor::vulkan_renderer {
 
-ImGUIOverlay::ImGUIOverlay(const wrapper::Device &device, const wrapper::Swapchain &swapchain)
+ImGUIOverlay::ImGUIOverlay(const wrapper::Device &device, const wrapper::Swapchain &swapchain,
+                           RenderGraph *render_graph, TextureResource *back_buffer)
     : m_device(device), m_swapchain(swapchain) {
-    assert(device.device());
```

Comment:
I assume you removed the assertions because the data is externally verified to be in a valid state already when passed into here?